<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QDialog" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>230</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="cellullar.qrc">:/img/about.png</pixmap>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCredits">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Credits</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLicense">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>License</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>121</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#7070ff;</string>
   </property>
   <property name="text">
    <string>&lt;a href=&quot;https://github.com/iwvq/cellullar&quot;&gt;github.com/iwvq/cellullar&lt;/a&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>240</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="cellullar.qrc">:/img/app_64.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="labelTitle">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#404040;</string>
         </property>
         <property name="text">
          <string>Cellullar v0.1a</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelBuildOn">
         <property name="styleSheet">
          <string notr="true">color:#404040;</string>
         </property>
         <property name="text">
          <string>Build on Oct 22 2023 05:08:23.442</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCopying">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#404040;</string>
         </property>
         <property name="text">
          <string>Copyright (c) 2023 Ivar Vilca</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButtonCredits</zorder>
  <zorder>pushButtonLicense</zorder>
  <zorder>label_6</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>textEdit</zorder>
 </widget>
 <resources>
  <include location="cellullar.qrc"/>
 </resources>
 <connections/>
</ui>
